File: Assignment_40/program2.c
//  Write a recursive program which display below pattern.   
 // Output :  1 2 3 4 5

 
///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.c
// Description :  Recursive program which display below pattern.
// Autor       :  Ankita Anil Patil
// Date        :  23/07/2025
///////////////////////////////////////////////////////////////////////////////

  

#include<stdio.h>

void Display() 
{ 
    static int i = 1;

    if(i <= 5)
    {
         printf("%d\t",i);
         i++;
         Display();
    }

} 
int main() 
{ 
    Display(); 
    return 0; 
} 

File: Assignment_49/program2.java
//  Write Java program which accept String from user and display below pattern. 
/*
    Input : Hello
    Output: H e l l o
            H e l l *
            H e l * *
            H e * * *
            H * * * *
 */


  
///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.java
// Description :  Accept String from user and display below pattern.              
// Autor       :  Ankita Anil Patil
// Date        :  19/08/2025
///////////////////////////////////////////////////////////////////////////////


 import java.util.Scanner;

class Pattern 
{ 
    public void Pattern(String str) 
    { 
        char[] strnew = str.toCharArray();

        int i = 0, j = 0;
        for(i = strnew.length; i > 0 ; i--)
        {
            for(j = 0; j < strnew.length; j++)
            {
                if( i > j)
                {
                    System.out.print(strnew[j] +"\t");
                    
                }
                else
                {
                    System.out.print("*" +"\t");
                    
                }
                
            }

            System.out.println();
        }

    } 
}

class program2
{
    public static void main(String[] A) 
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter string");
        String str = sobj.nextLine();

        Pattern pobj = new Pattern();
        pobj.Pattern(str);
        
    }
    
}